src/app/components/study-accordion/study-accordion.directive.js

```diff
@@ -64,4 +64,11 @@ angular.module('transmartBaseUi')
         }
       };
 
+      $scope.clearMetadata = function () {
+        $scope.metadataObj = {};
+        var query = document.getElementsByClassName("popover ng-isolate-scope right fade in");
```

Comment:
actually not sure, I haven't tried it with fewer classes, thought the more classes we add to the selector, the less probable we accidentally select some elements that we do not want to remove (no id selection, bummer...). Would this (i.e. using all these classes) be a caveat for potential refactoring? or performance issue?